.github/workflows/automerge.yml

```diff
@@ -1,5 +1,7 @@
 name: Auto merge
-on: workflow_call
+on:
+  - pull_request_target
```

Comment:
I do not think we need this change in the shared workflow file. It needs to be in the calling workflow, not in the shared. If you want to have the shared workflow in the .github repository for automerge then as we agreed we need a local directory under workflows and you would place the calling automerge.yml there.
So I suggest we cancel this PR.

Reply:
I think instead of the above we need to create automerge.yml in the workflows/local directory with the following contents:

name: Auto merge
on:
  pull_request_target

jobs:
  auto-merge:
    permissions:
      pull-requests: write
      contents: write
    uses: enarx/.github/.github/workflows/automerge.yml@main